src/chapters/appendix-proofstyle.tex

```diff
@@ -150,17 +150,17 @@
 	it must be clear to your reader whether you mean ``for all $x\in S$''
 	or just ``for some $x\in S$.''
 	If you just say something like ``$y=x^2$ where $x\in S$,''
-	the word ``where'' doesn't indicate whether you mean ``for all'' or ``some''.
+	the word ``where'' doesn't indicate whether you mean ``for all'' or ``some.''
 	
 	Phrases indicating the quantifier ``for all'' include 
-	   ``Let $x\in S$'';
-	   ``for all $x\in S$'';
-	   ``for every $x\in S$'';
-	   ``for each $x\in S$''; etc.
+	   ``Let $x\in S$;''
```

Comment:
Semicolons inside make this confusing.  The phrase is `Let x in S` not `Let x in S;`.  American English got this one wrong.